### PR TITLE
Feature/cleanup code

### PR DESCRIPTION
**Pull request by cschwinderg**
This pull request mainly makes the app and etl sides a little bit more consistent. I'm sorry about the size of this pull request. I made the prettier settings more consistent and ran prettier on the app/server files.

## Related Issues:
* [EQ-89](https://jira.epa.gov/browse/EQ-89)

## Main Changes:
* ab6a05f147afbeb86fe8f7a3693a8f1539ace88a - Made prettier settings consistent across the app and ran prettier on the app/server files. 
* 93531cffa81f974da483bc80b6797181067734f2 - Updated the app/server code to use ES6 module imports like we do in the etl side. Also modified the logger code (both app/server and etl) so that we don't have to alias it everytime (i.e. `const log = logger.logger;`). 
* 06a9a747222358d073540e584d12864bdd370513 - Moved the `mapping` variable out to a separate file `tableConfig.js` for both the app/server and etl sides. Also, put a note at the top to remind us to keep the two `tableConfig` files synced up.
  * I had thought about moving this to S3, but I'm worried about security implications of storing this in a public bucket. Our client side doesn't have access to the private bucket and I think we want to keep it that way. If you want, we can discuss this more during our EQ checkin call next week.
* e6ff888189a4f37d961f0d0edf235a469d6f5942 - Updated the index creation code to utilize the new `tableConfig.js` file.
* f50f786923ef18c82a65950df240dec5ea2cacb9 - Removed the `CREATE_INDEXES_AFTER_DATA` environment variable. We will always want to create indexes after loading the data. The overall etl time is about the same either way, but if the indexes are created first, then the indexes are more fragmented which impacts query performance.
* 247ae831677a2fb2e804be5e03c9a39ed73afb07 - Fixed a bug where the etl crashed if the `ready.json` file, in the S3 bucket, has a status of `nogo`. 

## Steps To Test:
1. Run the etl
2. Verify all tables, indexes and data are loaded. 
  * Note: I did add a few more indexes to the assessments table, because I noticed some more columns I missed.
3. Run the app and verify everything still works.
4. You can also run cypress tests, just be aware that you will need to fix the tests like how Justin did in PR #48 . I avoided changing cypress tests to avoid merge conflicts.

## TODO:
- [x] Rerun prettier on the app/cypress files. There is an open PR with changes to cypress code so I skipped that.
- [ ] Consider removing the stream national downloads code. I left this in for now, in case we want to use it for the national downloads page. If we end up needing it, we can remove it when we create the national downloads page.

### CODE REVIEW

**Comment by cschwinderg** [comment]
Please hold off on merging this PR in. I just want to verify the ETL status prior to merging this in.

File: etl/app/server/database.js
Comment (maxdiebold-erg) [inline etl/app/server/database.js:587]: This is causing a warning in the PostgreSQL logs. Since we're not executing each `profileEtl` as a transaction, and instead aborting the ETL upon any error, we can remove this.

**Review by maxdiebold-erg — CHANGES_REQUESTED** [review]
After merging in #48, I went ahead and ran prettier on the cypress files. 

I removed the `indexTableName` variables, that were leftover from before refactoring the `buildIndexQuery` function.

Also, I removed some unnecessary imports and fixed a couple smells.

There is one line that is throwing warning messages in the PostgreSQL logs, which I noted below. I didn't commit that change, since I thought you'd want to check it out first.

I haven't re-run the cypress tests yet. A few failed when I ran them on the last PR, but I think it was due to different local database contents. I'll try again and make sure.

**Comment by cschwinderg** [comment]
@maxdiebold-erg Good catches and thank you for making those changes!

**Review by maxdiebold-erg — APPROVED** [review]

File: etl/app/server/database.js
Comment (cschwinderg) [reply etl/app/server/database.js]: I changed this so that we fire off individual queries for creating the indexes. This just makes it so that we can check on the progress of indexes being created for a table. If we just remove the `COMMIT` line, then we can't see any indexes for a table until all of them have been created.

**Pull request merged**
